website/source/docs/connect/mesh_gateway.html.md

```diff
@@ -33,6 +33,7 @@ Mesh gateways also require that your Consul datacenters are configured correctly
 - Each of your [datacenters](/docs/agent/options.html#datacenter) must have a unique name. 
 - Your datacenters must be [WAN joined](https://learn.hashicorp.com/consul/security-networking/datacenters).
 - The [primary datacenter](/docs/agent/options.html#primary_datacenter) must be set to the same value in both datacenters. This specifies which datacenter is the authority for Connect certificates and is required for services in all datacenters to establish mutual TLS with each other.
+- Currently, Envoy is the only proxy with Mesh Gateway capabilities. 
```

Comment:
Should we be more explicit and say that any service that needs to send traffic outwards through a mesh gateway will need to be running the Envoy sidecar proxy?

For ingress, I think the built-in sidecar proxy will work?